views/main.html

```diff
@@ -18,17 +18,7 @@
         <script src="https://use.fontawesome.com/2155f7b139.js"></script>
     </head>
     <body id="bootstrap-override">
-        <div id="web-monitoring-ui-root">
-            <div class="row">
-                <div class="col-md-12">
-                    <div class="progress active progress-striped">
-                        <div class="progress-bar progress-success" style="width:100%" role="progressbar" aria-valuemin="0" aria-valuemax="100" aria-valuenow="100">
-                            <span>Please wait<span class="dotdotdot"></span></span>
-                        </div>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <div id="web-monitoring-ui-root"></div>
```

Comment:
There is a short amount of time before the React app renders, so I was suggesting that a `div.loading` was placed in the root div where the other one was before. It gets quickly replaced anyway, so not a big deal, just me being fussy. I'm working on some loading stuff and I really want to use your loading component, so I'll merge and just add this myself.